gap/PolygonalComplexes/animating.gi

```diff
@@ -88,6 +88,14 @@ InstallMethod( GetVertexCoordinates3DNC,
 RedispatchOnCondition(GetVertexCoordinates3DNC, true, [IsTwistedPolygonalComplex, IsPosInt, IsRecord],
     [IsSimplicialSurface], 0);
 
+InstallMethod( GetAllVertexCoordinates3DNC,
+    "for a simplicial surfaceand a record",
```

Comment:
A whitespace is missing here.

Reply:
See [772bfc6](https://github.com/gap-packages/SimplicialSurfaces/pull/259/commits/772bfc640c06f07b2470aadb228b7fdb9ca35aa5)